### ISSUE TITLE
PathAssign parsed with wrong field name

### ISSUE THREAD

**Issue by hypotext** [penrose/penrose#443]
Repro: same as #441

Expected: `PathAssign` values have fields `tag`, `type`, `path`, `value` (in `types.d.ts`)

Got: `PathAssign` values have field `expr` instead of `value` (See JSON below)

Suggestion: Change parser to parse `PathAssign` values with field of name `value`, not `expr`

@wodeni Can you do a once-over on the parser to make sure all other expressions are parsed into things with the field names that match the grammar? This is just the first error I ran into; I don't know if there are, e.g. recurring problems where all fields named `value` are parsed into field of name `expr`.

---

```
{
                        "start": {
                            "line": 6,
                            "col": 2
                        },
                        "end": {
                            "line": 6,
                            "col": 32
                        },
                        "tag": "PathAssign",
                        "type": {
                            "start": {
                                "line": 6,
                                "col": 2
                            },
                            "end": {
                                "line": 6,
                                "col": 8
                            },
                            "tag": "TypeOf",
                            "contents": {
                                "start": {
                                    "line": 6,
                                    "col": 2
                                },
                                "end": {
                                    "line": 6,
                                    "col": 8
                                },
                                "tag": "Identifier",
                                "value": "scalar",
                                "type": "type-keyword"
                            }
                        },
                        "path": {
                            "start": {
                                "line": 6,
                                "col": 9
                            },
                            "end": {
                                "line": 6,
                                "col": 24
                            },
                            "tag": "LocalVar",
                            "contents": {
                                "start": {
                                    "line": 6,
                                    "col": 9
                                },
                                "end": {
                                    "line": 6,
                                    "col": 24
                                },
                                "tag": "Identifier",
                                "value": "vectorSpaceSize",
                                "type": "identifier"
                            }
                        },
                        "expr": {
                            "start": {
                                "line": 6,
                                "col": 27
                            },
                            "end": {
                                "line": 6,
                                "col": 32
                            },
                            "tag": "Fix",
                            "contents": 350
                        }
                    },
```

**Comment by hypotext** [comment]
Same issue happens for paths tagged `Override` (they have an `expr` field and no `value`)

**Comment by wodeni** [comment]
Good catch. To explain why these are sometimes hard to catch: the parsers are written in a custom `.ne` file format, so the IDE won't catch type mismatch until you compile them into `.ts` files. I went through the parser again and annotated all postprocessors with return types, and then inspected the compiled `.ts` file to knock out all type errors. 

Fixed in c952679.
